temporian/core/operators/glue.py

```diff
@@ -201,7 +201,7 @@ def glue(
     Returns:
         EventSet with concatenated features.
     """
-    if len(inputs) == 1:
+    if len(inputs) == 1 and isinstance(inputs[0], EventSetNode):
```

Comment:
This fixes an unrelated bug which caused glue(literally anything) to return that same thing, discovered while passing `glue(evset_list)` instead of `glue(*evset_list)`

Reply:
Updated combine too